pkg/arvo/gen/who.hoon

```diff
@@ -0,0 +1,28 @@
+/+  *server
+::  @p request handler
+::
+|=  [[now=@da eny=@ bek=beak] ~ ~]
+|=  [authorized=? =request:http]
+^-  simple-payload:http
+=/  url=(unit pork:eyre)
+  (rush url.request apat:de-purl:html)
+::
+::  url doesn't match expected binding from :launch
+::
+?.  ?=(^ url)
+  ~|  [%invalid-url url.request]  !!
+?.  ?=([%who *] q.u.url)
+  ~|  [%unknown-url url.request]  !!
+::
+::
+?.  ?=(~ t.q.u.url)
+  [[%404 ~] ~]
+=/  response=json
+  (frond:enjs:format %who (ship:enjs:format p.bek))
```

Comment:
The minimalist in me wants to just send `(scow %p p.bek)` over the wire as-is instead of a neat json bundle, but I guess this is slightly more palatable to clients?

Reply:
Sending just the string makes any change to the payload a breaking change, so I'd rather not. But up to you